perf/istio-install/setup_istio.sh

```diff
@@ -151,8 +141,8 @@ function install_istio() {
           "$WD/setup_prometheus.sh" ${DIRNAME}
       fi
       if [[ -n "${MIXERLESS}" ]]; then
-        kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
-        kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/stats_filter.yaml
+        kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/istio/master/tests/integration/telemetry/stats/prometheus/testdata/metadata_exchange_filter.yaml
```

Comment:
As we know these are version sensitive now. Is this going to work with both master and 1.4 of tools?

Reply:
this is legacy part now, I would remove it